chain/cmd/provider/main.go

```diff
@@ -21,6 +23,7 @@ import (
 )
 
 var txSender cmtx.TxSender
+var allowedCommands = map[string]bool{"curl": true, "ping": true}
```

Comment:
Example of the workaround you need to do when a language is so stupid that it does not have `set`